scripts/typeahead.js

```diff
@@ -161,6 +161,12 @@ const searchOptions = [
     type: 'Pokérus',
     page: '',
   },
+  // Dream Orbs
+  {
+    display: 'Dream Orbs',
+    type: 'Dream Orbs',
```

Comment:
This is the correct way of adding it, but you will need to remove the space as the page is `DreamOrbs` or rename the folders to be `Dream Orbs`
```suggestion
    display: 'DreamOrbs',
    type: 'DreamOrbs',
```

Reply:
I think the folder should be `Dream Orbs`. The other pages like Hatchery Helpers also use spaces.

Reply:
I don't think I can change the folder name easily from my device...

Reply:
No worries, i'll update it into this PR